From: Simon Frankenberger <simon-alpine@fraho.eu>
Subject: Run tests in headless mode

Could't figure out how to make them run with xvfb-run, so just disable the ui tests

diff -ru old/test/basic/Basic.gmk new/test/basic/Basic.gmk
--- old/test/basic/Basic.gmk	2023-03-04 00:55:23.000000000 +0000
+++ new/test/basic/Basic.gmk	2023-04-25 13:41:38.485264258 +0000
@@ -55,7 +55,7 @@
 
 #----------------------------------------------------------------------
 
-ENVVARS="DISPLAY=$${DISPLAY:-`uname -n`:0.0}"
+ENVVARS="-Djava.awt.headless=true"
 
 ifeq ($(OS_NAME), windows)
   BASIC_TESTS := $(shell cygpath -m $(abspath $(TESTDIR)/share/basic) )
